modules/mono/editor/GodotTools/GodotTools/BuildTab.cs

```diff
@@ -41,7 +41,7 @@ private class BuildIssue : Reference // TODO Remove Reference once we have prope
         public bool ErrorsVisible { get; set; } = true;
         public bool WarningsVisible { get; set; } = true;
 
-        public Texture2D IconTexture
+        public Texture IconTexture
```

Comment:
Why is this needed? AFAIK `GetIcon` returns `Texture2D` and the only place `IconTexture` is used is being passed as a parameter that takes `Texture2D` as well. In fact, this change would break the latter.